packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -50,6 +50,47 @@ void main() {
     expect(find.byTooltip('Add'), findsOneWidget);
   });
 
+  // Regression test for: https://github.com/flutter/flutter/pull/21084
+  testWidgets('Floating Action Button tooltip (long press button edge)', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new MaterialApp(
+        home: const Scaffold(
+          floatingActionButton: FloatingActionButton(
+            onPressed: null,
+            tooltip: 'Add',
+            child: Icon(Icons.add),
+          ),
+        ),
+      ),
+    );
+
+    expect(find.byType(Text), findsNothing);
+    await tester.longPressAt(_rightEdgeOfFab(tester));
+    await tester.pumpAndSettle();
+
+    expect(find.byType(Text), findsOneWidget);
```

Comment:
This is OK, since if there's one Text widget, it's probably the tooltip. However `find.text('Add')` would be a little better (here and below).

Reply:
Thanks for the tip, done.